modules/021-cni-cilium/images/cilium/Dockerfile

```diff
@@ -61,32 +63,27 @@ COPY --from=rootfs / /
 
 # cilium-envoy from github.com/cilium/proxy
 #
-FROM quay.io/cilium/cilium-envoy:9c0d933166ba192713f9e2fc3901f788557286ee@sha256:943f1f522bdfcb1ca3fe951bd8186c41b970afa254096513ae6e0e0efda1a10d as cilium-envoy
+# https://github.com/cilium/cilium/blob/v1.11.12/images/cilium/Dockerfile#L6
+FROM quay.io/cilium/cilium-envoy:3b70fad0b9514720f33db82841907821202c1f02@sha256:8cca16ce66a0960a207cbf518ee2e0d923ae2a49207b154cdc37c2d95f583180 as cilium-envoy
 
 #
 # Hubble CLI
 #
 FROM ${CILIUM_BUILDER_IMAGE} as hubble
-RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.11.5.tar.gz | tar xvz -C /tmp/cilium-repo
-RUN bash /tmp/cilium-repo/cilium-1.11.5/images/cilium/download-hubble.sh
+RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.11.12.tar.gz | tar xvz -C /tmp/cilium-repo
+RUN bash /tmp/cilium-repo/cilium-1.11.12/images/cilium/download-hubble.sh
 RUN /out/linux/amd64/bin/hubble completion bash > /out/linux/bash_completion
 
 FROM ${CILIUM_BUILDER_IMAGE} as builder
 
 RUN apt-get update && apt-get install patch -y
 
-RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.11.5.tar.gz | tar xvz -C /tmp/cilium-repo
-WORKDIR /tmp/cilium-repo/cilium-1.11.5
+RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.11.12.tar.gz | tar xvz -C /tmp/cilium-repo
```

Comment:
Can you fix the double-wget thingy? Previous one is on line 73. You can copy it between stages. I've done this in a hurry during the initial implementation.

Reply:
I did this for 1.12.5, ok i will fix it here too